Service/ZGWService.php

```diff
@@ -27,46 +27,63 @@ public function __construct(EntityManagerInterface $entityManager, ParameterBagI
         $this->parameterBag = $parameterBag;
     }
 
-    /*
-   * Returns a welcoming string
-   *
-   * @return array
-   */
+    /** 
+     * Returns a updated zaak with a new besluit.
+     * 
+     * @param array $data 
+     * @param array $configuration 
+     *
+     * @return array Zaak.
+     */
     public function postZaakBesluitHandler(array $data, array $configuration): array
     {
         $this->data = $data;
         $this->configuration = $configuration;
+        $method = $this->data['parameters']->getMethod();
+
+        switch ($method) {
+            case 'POST':
+            case 'PUT':
+            case 'PATCH':
+                $explodedArray = explode('/api/zrc/v1/zaken/', $this->data['parameters']->getPathInfo());
+                $explodedZaakId = explode('/zaakbesluiten', $explodedArray[1]);
```

Comment:
In installation.json you just changed this to /besluiten

Reply:
I shall fix that, weird that it currently works